controllers/usersController.js

```diff
@@ -48,6 +48,29 @@ const getUser = async (req, res) => {
   }
 }
 
+/**
+ * Checks whether a user exists or not
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const checkUser = async (req, res) => {
+  try {
+    const result = await userQuery.fetchUser({ username: req.params.username })
+    if (!result.userExists) {
+      return res.json({
+        message: 'User name Available',
+        userExists: result.userExists
```

Comment:
Do we need to return the data of the user with that username too? 🤔

Reply:
we are not returning data of user.it will return boolean false. Shall I remove 'userExists' Property and return only message in case of success?